libgrive/src/json/Val.hh

```diff
@@ -138,6 +138,7 @@ template <> struct Val::Type2Enum<double>		{ static const TypeEnum type = double
 template <> struct Val::Type2Enum<std::string>	{ static const TypeEnum type = string_type ; } ;
 template <> struct Val::Type2Enum<Val::Array>	{ static const TypeEnum type = array_type ; } ;
 template <> struct Val::Type2Enum<Val::Object>	{ static const TypeEnum type = object_type ; } ;
+template <> struct Val::Type2Enum<std::vector<std::string> >	{ static const TypeEnum type = string_list_type ; } ;
```

Comment:
in fact I think it doesn't really matter because the proper place for move operation arguments is just in Drive::Move() function arguments :-) and it should also be simpler to implement :-)